packages/console/src/components/Drawer/index.module.scss

```diff
@@ -0,0 +1,28 @@
+@use '@/scss/underscore' as _;
+
+.content {
+  position: absolute;
+  left: 50%;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  outline: none;
+  background: var(--color-card-background);
+  padding: _.unit(6);
+  overflow-y: auto;
```

Comment:
nit: consider adding slide animation

Reply:
Created LOG-1891 to track this improvement.